lib/appliance_console/database_configuration.rb

```diff
@@ -118,7 +118,22 @@ def ask_for_database_credentials
       self.host     = ask_for_ip_or_hostname("database hostname or IP address", host) if host.blank? || !local?
       self.database = just_ask("name of the database on #{host}", database) unless local?
       self.username = just_ask("username", username) unless local?
-      self.password = ask_for_password_or_none("database password on #{host}", password)
+      count = 0
+      loop do
+        count += 1
+        password1   = ask_for_password_or_none("database password on #{host}", password)
+        # if they took the default, just bail
+        break if (password1 == password)
+        password2   = ask_for_password("database password again")
```

Comment:
Looks good, I'd maybe want to see what the prompts look like.  Usually they ask you to verify your password but this looks good @kbrock